frontend/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM nginx:alpine
-COPY nginx.conf /etc/nginx/nginx.conf
+FROM nginx
+COPY nginx.conf /etc/nginx/conf.d/default.conf
 WORKDIR /usr/share/nginx/html
 COPY dist/frontend .
-EXPOSE 4200
\ No newline at end of file
+CMD sed -i -e 's/$PORT/'"$PORT"'/g' /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'
```

Comment:
what does it do?

Reply:
This way we are setting $PORT in nginx.conf

Reply:
isn't that constant? 80? or 443? always the same?

Reply:
No, it's always a different one.